lib/ret_web/channels/hub_channel.ex

```diff
@@ -507,6 +486,16 @@ defmodule RetWeb.HubChannel do
     {:noreply, socket}
   end
 
+  def handle_out("hub_refresh" = event, %{stale_fields: stale_fields} = payload, socket) do
+    push(socket, event, payload)
+
+    if stale_fields |> Enum.member?("member_permissions") do
```

Comment:
This was a bit confusing at first glance, maybe add comment:
`# If hub member permissions change, everyone should flush their new permissions into presence`